### ISSUE TITLE
No results on tpc-c 10k plus stuck fetching logs 

### ISSUE THREAD

**Issue by awoods187** [cockroachdb/cockroach#31788]
**Describe the problem**

After completing the restore and setting up the data set, TPC-C did not produce any results on a 21 node 16 cpu cluster with increased ramp.

I see setting up the dataset:

`[ 710] tpccbench/nodes=21/cpu=16: setting up dataset (11h48m24s)`

Then no results:

```
> /Users/andrewwoods/go/bin/roachprod get andy-1540329660-tpccbench-nodes-21-cpu-16 logs artifacts/tpccbench/nodes=21/cpu=16/logs
andy-1540329660-tpccbench-nodes-21-cpu-16: getting (scp) logs artifacts/tpccbench/nodes=21/cpu=16/logs
.......
   1: done
   2: done
   3: done
   4: done
   5: done
   6: done
   7: done
   8: done
   9: done
  10: done
  11: done
  12: done
  13: done
  14: done
  15: done
  16: done
  17: done
  18: done
  19: done
  20: done
  21: done
  22: ~ scp -r -C -i /Users/andrewwoods/.ssh/id_rsa -i /Users/andrewwoods/.ssh/google_compute_engine -o StrictHostKeyChecking=no andrewwoods@35.231.23.226:logs artifacts/tpccbench/nodes=21/cpu=16/22.logs
scp: logs: No such file or directory
: exit status 1
2018/10/24 05:11:07 get logs failed
failed to download logs: /Users/andrewwoods/go/bin/roachprod get andy-1540329660-tpccbench-nodes-21-cpu-16 logs artifacts/tpccbench/nodes=21/cpu=16/logs returned:
stderr:
2018/10/24 05:11:07 get logs failed

stdout:
andy-1540329660-tpccbench-nodes-21-cpu-16: getting (scp) logs artifacts/tpccbench/nodes=21/cpu=16/logs
.......
   1: done
   2: done
   3: done
   4: done
   5: done
   6: done
   7: done
   8: done
   9: done
  10: done
  11: done
  12: done
  13: done
  14: done
  15: done
  16: done
  17: done
  18: done
  19: done
  20: done
  21: done
  22: ~ scp -r -C -i /Users/andrewwoods/.ssh/id_rsa -i /Users/andrewwoods/.ssh/google_compute_engine -o StrictHostKeyChecking=no andrewwoods@35.231.23.226:logs artifacts/tpccbench/nodes=21/cpu=16/22.logs
scp: logs: No such file or directory
```
Then, the cluster is stuck fetching logs.
```
[ 954] tpccbench/nodes=21/cpu=16: fetching logs (4h4m0s)
[ 955] tpccbench/nodes=21/cpu=16: fetching logs (4h5m0s)
```

**To Reproduce**

```diff
diff --git a/pkg/cmd/roachtest/tpcc.go b/pkg/cmd/roachtest/tpcc.go
index b6fee081e3..df14ae6ae6 100644
--- a/pkg/cmd/roachtest/tpcc.go
+++ b/pkg/cmd/roachtest/tpcc.go
@@ -403,7 +403,7 @@ func loadTPCCBench(
        }
 
        partArgs := ""
-       rebalanceWait := time.Duration(b.LoadWarehouses/100) * time.Minute
+       rebalanceWait := time.Duration(b.LoadWarehouses/25) * time.Minute
        switch b.LoadConfig {
        case singleLoadgen:
                t.l.Printf("splitting and scattering\n")
@@ -677,11 +677,11 @@ func registerTPCCBench(r *registry) {
                        // StoreDirVersion: "2.0-5",
                },
                {
-                       Nodes: 3,
-                       CPUs:  16,
+                       Nodes: 21,
+                       CPUs:  16,
```

`bin/roachtest bench '^tpccbench/nodes=21/cpu=16$$' --wipe=false --user=andy`

**Environment:**
 - CockroachDB version 2.1 rc 1

When I ran the following command:
```
root@localhost:26257/defaultdb> select * from crdb_internal.gossip_alerts
                             -> ;
  node_id | store_id | category |            description             | value  
+---------+----------+----------+------------------------------------+-------+
        4 |        3 | metrics  | queue.raftsnapshot.process.failure |   170  
       14 |       14 | metrics  | queue.raftsnapshot.process.failure |    30  
        9 |        9 | metrics  | queue.raftsnapshot.process.failure |    10  
        2 |        2 | metrics  | queue.raftsnapshot.process.failure |   200  
       20 |       20 | metrics  | queue.raftsnapshot.process.failure |   100  
       10 |       10 | metrics  | queue.raftsnapshot.process.failure |    10  
       16 |       16 | metrics  | queue.raftsnapshot.process.failure |    30  
        3 |        4 | metrics  | queue.raftsnapshot.process.failure |    20  
        1 |        1 | metrics  | queue.raftsnapshot.process.failure |   210  
       11 |       11 | metrics  | queue.raftsnapshot.process.failure |   101  
       13 |       13 | metrics  | queue.raftsnapshot.process.failure |   460  
       19 |       19 | metrics  | queue.raftsnapshot.process.failure |  6262  
       12 |       12 | metrics  | queue.raftsnapshot.process.failure |   670  
       21 |       21 | metrics  | queue.raftsnapshot.process.failure |    20  
       15 |       15 | metrics  | queue.raftsnapshot.process.failure |   130  
       17 |       17 | metrics  | queue.raftsnapshot.process.failure |    10  
        7 |        8 | metrics  | queue.raftsnapshot.process.failure |  1710  
       18 |       18 | metrics  | queue.raftsnapshot.process.failure |    50  
        6 |        6 | metrics  | queue.raftsnapshot.process.failure |  3750  
(19 rows)

Time: 3.232591ms
```
`andy-1540329660-tpccbench-nodes-21-cpu-16`
https://gist.github.com/awoods187/2ff7afc4f0c3caa871c4af65fe5f34ca

**Comment by tschottdorf** [comment]
Assuming merges are turned off, this should now work better on master.

**Comment by tschottdorf** [comment]
(The symptoms here closely match those in issues that seem to have been addressed by #31914, at least in my test runs. The test runner got stuck during teardown.)

I'll close this out.
